packages/rocketchat-ui/views/app/videoCall/videoCall.coffee

```diff
@@ -61,6 +61,28 @@ Template.videoCall.helpers
 			template.mainVideo.set '$auto'
 			return
 
+	mainVideoFlip: ->
+		template = Template.instance()
+		webrtc = WebRTC.getInstanceByRoomId(Session.get('openedRoom'))
+		myself = false;
+		localUrl = WebRTC.getInstanceByRoomId(Session.get('openedRoom')).localUrl.get()
+
+		if template.mainVideo.get() is '$self'
+			myself = true
+
+		if !myself && template.mainVideo.get() is '$auto'
+			remoteItems = webrtc.remoteItems.get()
+			if remoteItems?.length > 0
+				myself = remoteItems[0].url == localUrl
+
+			if !myself
+				myself = webrtc.localUrl.get() == localUrl
+
+		if !myself && webrtc.remoteItemsById.get()[template.mainVideo.get()]?
+			myself = webrtc.remoteItemsById.get()[template.mainVideo.get()].url == localUrl
+
+		return myself && !WebRTC.getInstanceByRoomId(Session.get('openedRoom')).screenShareEnabled.get()
```

Comment:
Any ideas how to reuse logic from mainVideoUrl. All I need to check is if currently shown `mainVideoUrl == selfVideoUrl`